### PR TITLE
FALCON-817: Passing groupid to oozie while scheduling the process

### PR DESCRIPTION
**Pull request by nperiwal**

### CODE REVIEW

File: oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java [hunk 1]
```diff
@@ -178,6 +182,7 @@ public void schedule(Entity entity, Boolean skipDryRun, Map<String, String> supp
                 }
 
                 //Do dryRun of coords before schedule as schedule is asynchronous
+                LOG.info("The properties passed to oozie are: {}", properties.stringPropertyNames().toString());
```
Comment (pallavi-rao) [inline oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java]: Make it debug.

File: oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java [hunk 2]
```diff
@@ -153,6 +153,10 @@ public void schedule(Entity entity, Boolean skipDryRun, Map<String, String> supp
         if (StartupProperties.isServerInSafeMode()) {
             throwSafemodeException("SCHEDULE");
         }
+        //Adding group information to pass to oozie
```
Comment (pallavi-rao) [inline oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java]: This ties it up to Unix groups. But, I guess it ok for now

Comment (nperiwal) [reply oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java]: ok

File: oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java (hunk 1 above)
Comment (nperiwal) [reply oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java]: Fixed.

**Comment by nperiwal** [comment]
Thanks @pallavi-rao

**Pull request merged**